utils/vibe/utils/dictionarylist.d

```diff
@@ -72,7 +72,7 @@ struct DictionaryList(VALUE, bool case_sensitive = true, size_t NUM_STATIC_FIELD
 		return ret;
 	}
 	/// ditto
-	FieldTuple[] toRepresentation() {
+	FieldTuple[] toRepresentation() const {
```

Comment:
Probably not needed after `auto ref in` -> `auto ref`

Reply:
True, but I'd still leave it, since it can't hurt here. Only the added `const` in the unit tests are problematic.

Reply:
ok, I've reverted added const in unittest I've overooked